include/ibamr/KrylovLinearSolverStaggeredStokesSolverInterface.h

```diff
@@ -89,6 +89,12 @@ class KrylovLinearSolverStaggeredStokesSolverInterface : public StaggeredStokesS
      */
     void setVelocityPoissonSpecifications(const SAMRAI::solv::PoissonSpecifications& U_problem_coefs);
 
+	/*!
+	 * \brief Set if velocity and pressure have nullspace.
+	 */
+	void setComponentsHaveNullspace(const bool has_velocity_nullspace,
```

Comment:
(1) For velocity, is a boolean sufficient, or do we need to be passing in nullspace vectors?
(2) setComponentNullspaces().